README.md

```diff
@@ -58,15 +43,13 @@ Clone the FCCSW repository: **replace \<username\> by your github username**
 
```

Comment:
nice catch! 

The FCCSW env var is used in the tutorial. 
https://twiki.cern.ch/twiki/bin/view/FCC/FccSoftwareFramework

I have no strong opinion on what to do with this variable, but it's important to make sure that our tutorials work.